### PR TITLE
Rename Auto-GPT to AutoGPT

### PR DESCRIPTION
**Pull request by merwanehamadi**
### Changes 🏗️
Rename to AutoGPT except a few places

### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [ ] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of Auto-GPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

### CODE REVIEW

File: autogpts/autogpt/.devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # To boot the app run the following:
-# docker-compose run auto-gpt
+# docker-compose run AutoGPT
```
Comment (Pwuts) [inline autogpts/autogpt/.devcontainer/docker-compose.yml:2]: This isn't right

Comment (Pwuts) [inline autogpts/autogpt/.devcontainer/docker-compose.yml]: This isn't right

File: autogpts/autogpt/BULLETIN.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
 # --------------
 🌎 *Official Website*: https://agpt.co.
 📖 *User Guide*: https://docs.agpt.co.
-👩 *Contributors Wiki*: https://github.com/Significant-Gravitas/Auto-GPT/wiki/Contributing.
+👩 *Contributors Wiki*: https://github.com/Significant-Gravitas/AutoGPT/wiki/Contributing.
```
Comment (Pwuts) [inline autogpts/autogpt/BULLETIN.md]:
```suggestion
👩 *Contributors Wiki*: https://github.com/Significant-Gravitas/Nexus/wiki/Contributing.
```

File: autogpts/autogpt/BULLETIN.md [hunk 3]
```diff
@@ -2,20 +2,20 @@
...
 # v0.4.7 RELEASE HIGHLIGHTS! 🚀
 # -----------------------------
 This release introduces initial REST API support, powered by e2b's agent 
 protocol SDK (https://github.com/e2b-dev/agent-protocol#sdk). 
 
 It also includes improvements to prompt generation and support 
-for our new benchmarking tool, Auto-GPT-Benchmarks
-(https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks).
+for our new benchmarking tool, AutoGPT-Benchmarks
+(https://github.com/Significant-Gravitas/AutoGPT-Benchmarks).
```
Comment (Pwuts) [inline autogpts/autogpt/BULLETIN.md]: Different repo; is this also being renamed?

File: autogpts/autogpt/poetry.lock [hunk 4]
```diff
@@ -284,7 +284,7 @@ abstract-singleton = "*"
 
 [package.source]
 type = "git"
-url = "https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template"
+url = "https://github.com/Significant-Gravitas/AutoGPT-Plugin-Template"
```
Comment (Pwuts) [inline autogpts/autogpt/poetry.lock:287]: Different repo; is this being moved too?

File: autogpts/autogpt/tests/unit/test_logs.py [hunk 5]
```diff
@@ -11,8 +11,8 @@
             "COMMAND = browse_website  ARGUMENTS = {'url': 'https://www.google.com', 'question': 'What is the capital of France?'}",
         ),
         (
-            "{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': 'https://github.com/Significant-Gravitas/Auto-GPT, https://discord.gg/autogpt und https://twitter.com/SigGravitas'}",
-            "{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': 'https://github.com/Significant-Gravitas/Auto-GPT, https://discord.gg/autogpt und https://twitter.com/SigGravitas'}",
+            "{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': 'https://github.com/Significant-Gravitas/AutoGPT, https://discord.gg/autogpt und https://twitter.com/SigGravitas'}",
```
Comment (Pwuts) [inline autogpts/autogpt/tests/unit/test_logs.py]:
```suggestion
            "{'Schaue dir meine Projekte auf github () an, als auch meine Webseiten': 'https://github.com/Significant-Gravitas/AutoGPT, https://discord.gg/autogpt und https://twitter.com/Auto_GPT'}",
```

File: autogpts/forge/tutorials/001_getting_started.md [hunk 6]
```diff
@@ -37,8 +37,8 @@ git clone https://github.com/Significant-Gravitas/Auto-GPT.git
 
 Once you have clone the project change your directory to the newly cloned project:
 ```bash
-# The name of the directory will match the name you gave your fork. The defualt is Auto-GPT
-cd Auto-GPT
+# The name of the directory will match the name you gave your fork. The defualt is AutoGPT
```
Comment (Pwuts) [inline autogpts/forge/tutorials/001_getting_started.md]:
```suggestion
# The name of the directory will match the name you gave your fork. The default is AutoGPT
```

File: benchmark/README.md [hunk 7]
```diff
@@ -8,17 +8,17 @@ Save time and money while doing it through smart dependencies. The best part? It
 
 ## Scores:
 
-<img width="733" alt="Screenshot 2023-07-25 at 10 35 01 AM" src="https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks/assets/9652976/98963e0b-18b9-4b17-9a6a-4d3e4418af70">
+<img width="733" alt="Screenshot 2023-07-25 at 10 35 01 AM" src="https://github.com/Significant-Gravitas/AutoGPT-Benchmarks/assets/9652976/98963e0b-18b9-4b17-9a6a-4d3e4418af70">
```
Comment (Pwuts) [inline benchmark/README.md]: Different repo; is this also being renamed?

File: benchmark/README.md [hunk 8]
```diff
@@ -8,17 +8,17 @@ Save time and money while doing it through smart dependencies. The best part? It
...
 
 - 1- [Beebot](https://github.com/AutoPackAI/beebot)
 - 2- [mini-agi](https://github.com/muellerberndt/mini-agi)
-- 3- [Auto-GPT](https://github.com/Significant-Gravitas/Auto-GPT)
+- 3- [Auto-GPT](https://github.com/Significant-Gravitas/AutoGPT)
 
 ## Detailed results:
 
-<img width="733" alt="Screenshot 2023-07-25 at 10 42 15 AM" src="https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks/assets/9652976/39be464c-c842-4437-b28a-07d878542a83">
+<img width="733" alt="Screenshot 2023-07-25 at 10 42 15 AM" src="https://github.com/Significant-Gravitas/AutoGPT-Benchmarks/assets/9652976/39be464c-c842-4437-b28a-07d878542a83">
```
Comment (Pwuts) [inline benchmark/README.md]: Same as on line 11

File: benchmark/agbenchmark/README.md [hunk 9]
```diff
@@ -57,7 +57,7 @@ https://github.com/smol-ai/developer/pull/114/files
 
 2- Create the submodule and the github workflow by following the same pattern as this example:
 
-https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks/pull/48/files
+https://github.com/Significant-Gravitas/AutoGPT-Benchmarks/pull/48/files
```
Comment (Pwuts) [inline benchmark/agbenchmark/README.md]: Different repo; is this also being renamed?

File: benchmark/agbenchmark/challenges/README.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# This is the official challenge library for https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks
+# This is the official challenge library for https://github.com/Significant-Gravitas/AutoGPT-Benchmarks
```
Comment (Pwuts) [inline benchmark/agbenchmark/challenges/README.md]: Different repo; is this also being renamed?

File: benchmark/frontend/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # agbenchmark-frontend
 
-Frontend for https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks
+Frontend for https://github.com/Significant-Gravitas/AutoGPT-Benchmarks
```
Comment (Pwuts) [inline benchmark/frontend/README.md]: Different repo; is this also being renamed?

File: docs/content/challenges/introduction.md [hunk 12]
```diff
@@ -1,35 +1,35 @@
...
+    We're slowly transitioning to agbenchmark. agbenchmark is a simpler way to improve AutoGPT. Simply run:
     
     ```
     agbenchmark
     ```
     
     and beat as many challenges as possible.
 
-For more agbenchmark options, look at the [readme](https://github.com/Significant-Gravitas/Auto-GPT-Benchmarks/tree/master/agbenchmark).
+For more agbenchmark options, look at the [readme](https://github.com/Significant-Gravitas/AutoGPT-Benchmarks/tree/master/agbenchmark).
```
Comment (Pwuts) [inline docs/content/challenges/introduction.md]: Different repo; is this also being renamed?

File: docs/content/plugins.md [hunk 13]
```diff
@@ -14,7 +14,7 @@ plugin_b:
   enabled: true
 ```
 
-See our [Plugins Repo](https://github.com/Significant-Gravitas/Auto-GPT-Plugins) for more info on how to install all the amazing plugins the community has built!
+See our [Plugins Repo](https://github.com/Significant-Gravitas/AutoGPT-Plugins) for more info on how to install all the amazing plugins the community has built!
 
-Alternatively, developers can use the [Auto-GPT Plugin Template](https://github.com/Significant-Gravitas/Auto-GPT-Plugin-Template) as a starting point for creating your own plugins.
+Alternatively, developers can use the [AutoGPT Plugin Template](https://github.com/Significant-Gravitas/AutoGPT-Plugin-Template) as a starting point for creating your own plugins.
```
Comment (Pwuts) [inline docs/content/plugins.md]: Different repos

**Review by Pwuts — CHANGES_REQUESTED** [review]

File: benchmark/agbenchmark/README.md (hunk 9 above)
Comment (waynehamadi) [reply benchmark/agbenchmark/README.md]: no, I am putting it back to Auto-GPT

**Review by Pwuts — APPROVED** [review]

**Pull request merged**